### ISSUE TITLE
Add a security login

### ISSUE THREAD

**Issue by ludoo0d0a** [azat-co/mongoui#43]
please intoduce a mininum of security.
at least, a basic authentication box can be enough (https://github.com/expressjs/basic-auth-connect).

Letting this app public is not safe. 

thanks

**Comment by azat-co** [comment]
@ludoo0d0a yes. sure! security is important... that's why just a basic auth might not be enough. the idea is that people will be forced to implement their own security on top of the app if we don't provide it... but what integration would you like to see? having a password will require a database for the app... which increases the complexity, hardcoding is another option.  GitHub OAuth will require getting GitHub keys, etc.
so it's not a simple path.
